models/models.app.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../db/connection');
+
+exports.selectAllTopics = () => {
+  return db
+    .query(
+      `
+  SELECT * FROM topics;
```

Comment:
Great code, just watch your indentation here 😄

Reply:
Yeah, I'll get that cleaned up. Thanks!